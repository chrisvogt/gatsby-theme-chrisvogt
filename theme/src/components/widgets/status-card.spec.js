import React from 'react'
import renderer from 'react-test-renderer'
import StatusCard from './status-card'

describe('StatusCard', () => {
  it('renders correctly', () => {
    const message = 'Lorum ipsum dolor sit amet.'
    const tree = renderer.create(<StatusCard>{message}</StatusCard>).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
