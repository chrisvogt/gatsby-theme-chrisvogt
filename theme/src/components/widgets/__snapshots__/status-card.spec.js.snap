// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StatusCard renders correctly 1`] = `
<div
  className="css-bmvaq1-Component"
/>
`;
